src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -78,6 +78,7 @@ class Crystal::Doc::Generator
 
     main_index = Main.new(raw_body, Type.new(self, @program), repository_name)
     File.write File.join(@output_dir, "index.json"), main_index
+    File.write File.join(@output_dir, "index.jsonp"), main_index.to_jsonp
```

Comment:
I think `search-index.js` (from Rust doc) is better than `index.jsonp`. The extension `jsonp` is unsound.

Reply:
Yeah, why JSONP? Still, there might a problem with loading `json` files over `file:///`.

Reply:
`search-index.js` seems like a good idea 👍